packages/mdc-toolbar/README.md

```diff
@@ -0,0 +1,121 @@
+# MDC Toolbar
+
+MDC Toolbar provides a RTL-aware Material Design toolbar component adhering to the
+[Material Design toolbar spec](https://www.google.com/design/spec/components/toolbars.html)
+Toolbars scroll with content by default, but supports fixed on
+top as well. Currently, this component does not yet support the "Waterfall" or
+"Flexible Header" patterns.
+
+
+## Installation
+
+```
+npm install --save @material/toolbar
+```
+
+
+## Usage
+
+Wrap the items with `mdc-toolbar` class in following way:
+
+```html
+<div class="mdc-toolbar">
+  <div class="mdc-toolbar__section">
+    <div class="mdc-toolbar__section mdc-toolbar__section--align-start">
+      <span class="mdc-toolbar__title">Title</span>
+    </div>
+  </div>
+</div>
+```
+
+### Fixed toolbars
+
+By default, toolbars scroll with the page content. To keep the toolbar fixed to
+the top of the screen, add an `mdc-toolbar--fixed` class to the toolbar element.
+
+**Body margin required**
+
+You need to wrap contents with `mdc-toolbar_fixed-container-adjust`
```

Comment:
SGTM. Acutally is this "You can" or "You should"? Should we be opinionated that this is the way we recommend?